src/App.jsx

```diff
@@ -14,33 +16,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const SORT_FIELD_BY_ALPHABET = 'alphabet';
+const SORT_FIELD_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+function sortedGoods(goods, { sortField }) {
+  const copiedGoods = [...goods];
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  switch (sortField) {
+    case SORT_FIELD_BY_ALPHABET:
+      copiedGoods.sort((good1, good2) => good1.localeCompare(good2));
+      break;
+
+    case SORT_FIELD_BY_LENGTH:
+      copiedGoods.sort((good1, good2) => good1.length - good2.length);
+      break;
+
+    default:
+      break;
+  }
+
+  return copiedGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+
+  const [reversedField, setReversedField] = useState(false);
```

Comment:
```suggestion
  const [isReversedField, setIsReversedField] = useState(false);
```
for boolean variables: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig